src/components/Offline/Offline.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const Offline = () => (
+  <div>
+    <header style={{ height: '84px' }}></header>
+    <main style={{ display: 'block', widht: '100%' }}>
+      <section class="section__block image__main">
+        <div class="wrapper" style={{ width: '800px', margin: 'auto' }}>
+          <h1 style={{ 'padding-bottom': '0' }}>Experimentamos dificultades técnicas.</h1>
+          <h2
+            style={{
+              'font-size': '18px',
+              'font-weight': '400',
+              'line-height': '1.2',
+              'letter-spacing': '0',
+              color: '#525845',
+              padding: '0 0 10px',
+            }}
+          >
+            ¡Lo sentimos! Volveremos a la normalidad lo antes posible.
```

Comment:
dont need english language?

Reply:
This page was only in spanish, I'm not sure @mrunge86 what do you think? And also if all is offline, it may include language support....